pkg/liqo-controller-manager/virtualNode-controller/namespaceMap_creation.go

```diff
@@ -40,10 +41,15 @@ func (r *VirtualNodeReconciler) createNamespaceMap(ctx context.Context, n *corev
 
 // ensureNamespaceMapPresence creates a new NamespaceMap associated with that virtual-node if it is not already present.
 func (r *VirtualNodeReconciler) ensureNamespaceMapPresence(ctx context.Context, n *corev1.Node) error {
+	// Only when the NamespaceMap is created for the first time it is necessary to check the presence of the local
+	// Technical namespace's name.
+	if err := r.checkLocalTechnicalNamespaceNamePresence(ctx, n.Annotations[liqoconst.RemoteClusterID]); err != nil {
```

Comment:
Is the annotation presence enforced or checked somewhere?

Reply:
yes it is checked in the reconcile here (https://github.com/liqotech/liqo/blob/b0edf16ea874ea5f6ce1aadb0ea90016d4484727/pkg/liqo-controller-manager/virtualNode-controller/virtualnode_controller.go#L66)